qml/NewCredentialView.qml

```diff
@@ -57,16 +60,38 @@ Flickable {
     }
 
     function scanQr() {
-        scanning = true
-        yubiKey.parseQr(ScreenShot.capture(), function (resp) {
-            scanning = false
-            if (resp.success) {
-                credential = resp
-            } else {
-                navigator.snackBarError(navigator.getErrorMessage(
-                                                                resp.error_id))
-            }
-        })
+        console.log("111")
```

Comment:
111oneone

Reply:
I think that is an outdated version of that file.

Reply:
Ah, yes, you are right. Sorry about that :)

/me fetches more coffee